src/workspace.coffee

```diff
@@ -550,7 +556,7 @@ class Workspace extends Model
   # Extended: Create a new text editor.
   #
   # Returns a {TextEditor}.
-  buildTextEditor: (params) ->
+  buildTextEditor_: (params) ->
```

Comment:
or capturing `this.buildTextEditor` as a local `buildTextEditor` before building the bound function